vegafusion-python/tests/test_jupyter_widget.py

```
@@ -461,7 +461,7 @@ def get_url():
         chrome_driver.execute_script(script)
 
         # Get canvas element (the canvas that Vega renders to)
-        @retry(wait=wait.wait_fixed(0.5), stop=stop.stop_after_delay(10))
+        @retry(wait=wait.wait_fixed(1.0), stop=stop.stop_after_delay(30))
         def get_canvas():
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `get_canvas`

(ANN202)